prow/markdown-index/test-infra-markdown-index-autobump-config.yaml

```diff
@@ -0,0 +1,23 @@
+# Code generated by rendertemplates. DO NOT EDIT.
```

Comment:
Does this file need to be autogenerated from templates? It doesn't contain any variables

Reply:
deleted